packages/opentelemetry-metrics/src/BaseObserverMetric.ts

```diff
@@ -30,7 +30,7 @@ const NOOP_CALLBACK = () => {};
  */
 export abstract class BaseObserverMetric extends Metric<BoundObserver>
   implements api.BaseObserver {
-  protected _callback: (observerResult: api.ObserverResult) => void;
+  protected _callback: (observerResult: api.ObserverResult) => unknown;
```

Comment:
not sure why you changed the void to a unknown ?

Reply:
because this might or not return something